<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenarioEditClass</class>
 <widget class="QMainWindow" name="ScenarioEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1350</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>想定编辑</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>      模型列表</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>1500</width>
      <height>1100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{border:4px solid rgb(200,200,255)} </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>60</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Pos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>60</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>60</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posXEdit">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="posYEdit">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="cmdEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>1200</y>
      <width>800</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mouseBackButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>鼠标恢复</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
